docs/scripts/api/buildComponentsDocumentation.ts

```diff
@@ -690,11 +664,17 @@ export default async function buildComponentsDocumentation(
           documentedInterfaces,
         });
 
+        if (response === null) {
+          return null;
+        }
+
+        const { name, imports, folder } = response;
+
         return {
           folderName: folder,
           pathname: `/x/api/${folder}/${kebabCase(name)}`,
           title: name,
-          plan: getPlan(packages),
+          plan: getPlan(imports),
```

Comment:
The only place where the `packages` were used is here to get the plan to display on the navbar.
We can do the exact same using the import strings.